pkg/network/driver/nft/nft.go

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Please consider introducing this driver/handler in a commit of its own (like the previous ones).